### PR TITLE
Adst 484

### PR DESCRIPTION
**Pull request by PiplaniVinay**

### CODE REVIEW

File: AlfrescoAuth/Sources/PKCE Authentication/AuthPkcePresenter.swift [hunk 1]
```diff
@@ -229,4 +336,60 @@ public class AuthPkcePresenter {
...
+            return
+        }
+        
+        let task = URLSession.shared.dataTask(with: url) { data, response, error in
+            if let error = error {
+                completion(.failure(error))
+                return
+            }
+            
+            guard let data = data else {
```
Comment (aman-alfresco) [inline AlfrescoAuth/Sources/PKCE Authentication/AuthPkcePresenter.swift:351]: Please correct the naming convention for **guard let data**

**Review by aman-alfresco — CHANGES_REQUESTED** [review]

File: AlfrescoAuth/Sources/PKCE Authentication/AuthPkcePresenter.swift [hunk 2]
```diff
@@ -229,4 +336,60 @@ public class AuthPkcePresenter {
...
                         error: error)
     }
+    
+    func fetchAppConfig(completion: @escaping (Result<OAuth2Data, Error>) -> Void) {
+        guard let url = URL(string: "\(self.configuration.baseUrl)/\(appConfig)") else {
+            return
+        }
+        
+        let task = URLSession.shared.dataTask(with: url) { data, response, error in
+            if let error = error {
```
Comment (gl-lovekesh) [inline AlfrescoAuth/Sources/PKCE Authentication/AuthPkcePresenter.swift:346]:
```suggestion
            if let err = error {
```

File: AlfrescoAuth/Sources/PKCE Authentication/AuthPkcePresenter.swift (hunk 1 above)
Comment (gl-lovekesh) [inline AlfrescoAuth/Sources/PKCE Authentication/AuthPkcePresenter.swift:351]:
```suggestion
            guard let resData = data else {
```

**Review by gl-lovekesh — CHANGES_REQUESTED** [review]
please check the suggestions

**Review by gl-lovekesh — APPROVED** [review]
LGTM

**Pull request merged**